app/gameboard_api.py

```diff
@@ -6,6 +6,7 @@
 from aiohttp_jinja2 import template
 
 from app.objects.c_ability import Ability
+from app.objects.secondclass.c_executor import Executor
```

Comment:
nit: imports should be lexicographically ordered within their respective group

Reply:
Check the updated order.